template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java

```diff
@@ -64,6 +64,11 @@ public static boolean isTagsResourceSupportedViaBlueprint(ClouderaManagerRepo cl
         return isVersionNewerOrEqualThanLimited(clouderaManagerRepoDetails::getVersion, CLOUDERAMANAGER_VERSION_7_1_0);
     }
 
+    public static boolean isRazConfigurationSupported(ClouderaManagerRepo clouderaManagerRepoDetails) {
+        LOGGER.info("ClouderaManagerRepo is compared for Raz Ranger support");
```

Comment:
should we check this in case of sdx app as well ? I mean based on the runtime we should not enable the raz configuration if runtime < 7.2.0